_includes/misc/supported_versions.md

```diff
@@ -4,7 +4,8 @@ Supported versions of {{site.productname}}:
 
 | Version |   Release Date    |  End of Support   |
 |:-------:|:-----------------:|:-----------------:|
-|   5.7   | February 24, 2021 |        TBD        |
+|   5.8   | TBD               |        TBD        |
```

Comment:
Just adding a reminder we need to add a date here before merging

- [ ] Add release date